msgraph-connector-demo/pom.xml

```diff
@@ -23,13 +25,48 @@
       </snapshots>
     </pluginRepository>
   </pluginRepositories>
+
   <build>
     <plugins>
+      <plugin>
+        <artifactId>maven-assembly-plugin</artifactId>
```

Comment:
I'd not pollute the root module pom.
but rather introduce a distinct sub-project/pom.xml ...
having a clear "-app" prefix. Containing the assembly instructions.
... and maybe the "deploy" part.

But if we mix it all up here, the maven build will get less flexible for future changes.